src/routes/docs/products/storage/quick-start/+page.markdoc

```diff
@@ -119,7 +119,34 @@ To upload a file, add this to you web, Flutter, Apple, or Android app.
 
   --cec8e8123c05ba25--
   ```
+  ```client-react-native
+  import { Client, Storage, ID, Permission } from 'react-native-appwrite';
 
+  const client = new Client()
+      .setEndpoint('https://cloud.appwrite.io/v1') // Your API Endpoint
+      .setProject('<PROJECT_ID>');                 // Your project ID
+
+  const storage = new Storage(client);
+
+  const promise = await storage.createFile(
+      '[BUCKET_ID]',
+      ID.unique(),
+      { 
+      name: 'file.jpeg', 
+      type: 'image/jpeg', 
+      uri: 'file:///data/user/0/host.exp.exponent/cache/ExperienceData/text.jpeg', 
+      size: 183799
+      }, // FILE
+      Permission.read('any')
```

Comment:
Make sure this uses the same params as all the other examples on this page. I think some other examples don't show permission, but if you want to add it here, make sure ALL examples are updated to be consistent!